stories/plugins/mention.stories.tsx

```diff
@@ -36,7 +36,7 @@ export const Example = () => {
       []
     );
 
-    const { target, setTarget, index, setIndex, setSearch, chars } = useMention(
+    const { MentionSelectComponent,  defaultOnChangeMention, defaultOnKeyDownMention } = useMention(
```

Comment:
I think `onChangeMention` and `onKeyDownMention` would be better names. Then you can delete the `onChangeMention.ts` and `onKeyDownMention.ts`

Reply:
I didn't want to introduce that big of a change, but if we are comfortable with breaking changes I'm happy to further simplify things

Reply:
Yes you can go ahead, we can introduce breaking change for any patch version until v1.0.